types/office-runtime/index.d.ts

```diff
@@ -18,10 +19,101 @@ Copyright (c) Microsoft Corporation
  * Office runtime namespace.
  */
 declare namespace OfficeRuntime {
+    /**
+     * Encapsulates methods that interacts with the Office UI
+     * @beta
+     */
+    interface UI {
+        /**
+         * Get the ribbon of the Office application.
+         */
+        getRibbon(): Promise<Ribbon>;
+    }
+    /**
+     * Encapsulates all the functionality provided to manage the state of the Office ribbon.
```

Comment:
Could this be simplified to "Encapsulates the management of the Office ribbon state." ?